### DISCUSSION TITLE
All subscription notifications have been completed, but there are still unread counts.

### DISCUSSION THREAD

**Discussion by izuolan** [community/community#45349] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

<img width="950" alt="image" src="https://user-images.githubusercontent.com/5241553/214604760-070aee4b-946a-483d-8e08-fd5dd6f70546.png">

This repository keeps getting an unread message, but nothing is actually showing up, which I think is a problem with the GitHub api.

**Comment by antfu** [comment]
I was annoyed by this for a while as well. Until I found that they have a `Mark All as Done` button that only shows when you have too many notifications. To make it shown, you can run the following code in your console and click the `Done` button to also dismiss the missing notification

```ts
document.querySelector('.js-notifications-mark-all-actions').removeAttribute('hidden')
```

**Reply by izuolan** [reply]
The reply to my Github ticket has temporarily alleviated my problem, although it still hasn’t been resolved until today. However, it no longer interferes with my notification bell count.

![image](https://github.com/community/community/assets/5241553/2796a01d-1f16-4362-aecd-a5a6eb32031d)

---

> In the interim, the curl command below will mark all of your notifications as read and clear the notification indicator.
> 
> A few things to note:
> 
> - In order to use this command a Personal Access Token (PAT) will be required and $TOKEN needs to be replaced by a personal access token you can create here: https://github.com/settings/tokens/new . [Creating a Personal Access Token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token#creating-a-token) also has the steps to create a token. After it is created, replace $TOKEN in the command with your new PAT.
> - Note: Please ensure that the PAT has the notifications scope.
> - Please update the date and time to your current date and note the time values must be in ISO 8601 format: YYYY-MM-DDTHH:MM:SSZ.
> 
> ```
> curl -H "Authorization: bearer $TOKEN" -X PUT -H "Accept: application/vnd.github.v3+json" https://api.github.com/notifications -d '{"last_read_at":"2022-03-03T11:55:00"}'
> ```
> 
> You can also export the current date and time in bash by running:
> 
> ```
> export now=$(date +%Y-%m-%dT%H:%M:%SZ)
> ```
> 
> Then you can use the date and time variable in the curl command as follows:
> 
> ```
> curl -H "Authorization: bearer $TOKEN" -X PUT -H "Accept: application/vnd.github.v3+json" https://api.github.com/notifications -d '{"last_read_at":"'"$now"'"}'
> ```
> 
> For your reference, here is the documentation for the [Notifications REST API endpoint](https://docs.github.com/en/rest/reference/activity#mark-notifications-as-read).

---

Until today, it has not been resolved. Perhaps they have already forgotten. 😂

![image](https://github.com/community/community/assets/5241553/2857b35b-c1c2-4350-8d30-b43b815288bb)
